README.md

```diff
@@ -347,4 +392,18 @@ class MyCustomApplicationLoader extends ApplicationLoader {
     (new GuiceApplicationLoader).load(newContext)
   }
 }
-```
\ No newline at end of file
+```
+
+#### Play 2.3
+
+```scala
+import play.api._
+import com.typesafe.conductr.bundlelib.play.Env
+
+object Global extends GlobalSettings {
+  val totalConfiguration = super.configuration ++ Configuration(Env.asConfig)
```

Comment:
I've created an issue for this: https://github.com/typesafehub/conductr-lib/issues/80